nexus-sdk/src/main/java/io/nexusrpc/Header.java

```diff
@@ -11,5 +11,11 @@ public class Header {
    */
   public static final String OPERATION_TIMEOUT = "Operation-Timeout";
 
+  /**
+   * Header for the unique ID returned by the StartOperation response for async operations. Must be
+   * set on callback headers to support completing operations before the start response is received.
+   */
+  public static final String HEADER_OPERATION_ID = "Nexus-Operation-Id";
```

Comment:
```suggestion
  public static final String OPERATION_ID = "Nexus-Operation-Id";
```

Figured we don't need to prefix since we didn't for the other two and they are in this class (but can add `NEXUS_` if you want, but I don't think it's needed)

Reply:
Remove `HEADER_`